uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -67,6 +71,7 @@ public static UiObject obtainUiObjectWithResourceId(@NonNull final String resour
     @NonNull
     public static UiObject obtainUiObjectWithResourceIdAndEnabledFlag(@NonNull final String resourceId,
                                                                       final boolean enabled) {
+        Logger.i(TAG, "Obtain an instance of an enabled UiObject for the resource Id..");
```

Comment:
let's also log the resource id and enabled value in the log message

Reply:
added.